javascript/MaterialXView/source/index.js

```diff
@@ -155,6 +156,13 @@ function init()
         viewer.getScene().setUpdateTransforms();
     });
 
+    setSceneLoadingCallback(file => {
```

Comment:
Set callback to do geometry load. Same as what's done from the menu but with the URI from the drop.